src/pages/workspaces/workspace/WorkspaceMenu.test.js

```diff
@@ -26,14 +25,6 @@ jest.mock('src/components/PopupTrigger', () => {
   };
 });
 
-// Mocking TooltipTrigger to avoid test environment issues with React Portal's requirement to use
-// DOM measure services which are not available in jest environment
-jest.mock('src/components/TooltipTrigger', () => ({
```

Comment:
Clickable no longer imports TooltipTrigger from src/components/TooltipTrigger, so this mock no longer had any effect on this test.